client/src/components/Comments/index.tsx

```diff
@@ -48,35 +59,89 @@ class Comments extends React.Component<ICommentsProps> {
         }
     }
 
+    public editComment = (comment: string, id: string, email: string): void => {
+        this.props.editCommentAsync(comment, id, email);
+    }
+
     public render(): JSX.Element {
         return (
-            <div className='flex-grow-1 comments border-top position-relative'>
-                <div className='position-absolute h-100'>
-                    {!!this.props.comments && this.props.comments.map((comment: any) => (
-                        <div className='one-comment px-3' key={comment._id}>
-                            <img
-                                src={comment.authorId.photoPath || noAvatar}
-                                alt='avatar'
-                                width={24}
-                                height={24}
-                                className='img-fluid rounded-circle mt-1 mr-1 mb-1'
-                            />
-                            <span className='mt-1'>{comment.authorId.username}</span>
-                            <div className='d-inline-flex mt-3 float-right edit-delete-comment'>
-                                <i className='fa fa-pencil mr-2 edit-comment'/>
-                                <i className='fa fa-trash-o delete-comment'/>
-                            </div>
-                            <p>{comment.comment}</p>
-                        </div>
-                    ))}
-                    <Waypoint
-                        scrollableAncestor={window}
-                        onEnter={(): void => {
-                            this.getMoreComments();
-                        }}
-                    />
+            <>
+                <div className='flex-grow-1 comments border-top position-relative'>
+                    <div className='position-absolute h-100'>
+                        {!!this.props.comments && this.props.comments.map((comment: any) => (
+                                <div className='one-comment px-3' key={comment._id}>
+                                    <img
+                                        src={comment.authorId.photoPath || noAvatar}
+                                        alt='avatar'
+                                        width={24}
+                                        height={24}
+                                        className='img-fluid rounded-circle mt-1 mr-1 mb-1'
+                                    />
+                                    <span className='mt-1'>{comment.authorId.username}</span>
+                                    {
+                                        comment.isEdit ?
```

Comment:
do we need to check it here too - this.props.user.email === comment.authorId.email?

Reply:
We check it on BE and on FE. Do we need it on FE ?